packages/yuudachi/locales/en-US/translation.json

```diff
@@ -4,7 +4,8 @@
 			"no_mod_role": "This command can only be used by moderators.",
 			"target_not_found": "The given member is not in this guild.",
 			"timed_out": "Action timer ran out.",
-			"no_mod_log_channel": "No mod log channel has been initialized yet."
+			"no_mod_log_channel": "No mod log channel has been initialized yet.",
+			"no_general_log_channel": "No general log channel has been initialized yet."
```

Comment:
What defines as a `general` log channel here?
The intent or rather the name is very unclear about it.

Reply:
I was thinking about `Reports` or `Anti-Raid Reports` but as per Souji, we can use it to store more than `Anti-Raid Reports`, as message bulk deletes

Reply:
But for that we already have the guild log channel 🤔

Reply:
Maybe he was talking about using the `.md` format and I misunderstood, I have some ideias for names:

- Archive Channel (Since it's used to store raid archives?)
- Anti-Raid Reports
- Anti-Raid Log

Reply:
The name is now `Anti-Raid Archive`